### ISSUE TITLE
sitting animation

### ISSUE THREAD

**Issue by MCLV-pandorabox** [mt-mods/homedecor_modpack#78]
love the sitting animation.
Butt, player keeps sitting after moving away from the chair

**Comment by wsor4035** [comment]
what game/mods/etc?

**Comment by SwissalpS** [comment]
I think he found this on pandorabox, meta repo at: https://github.com/pandorabox-io/pandorabox-mods

**Comment by MCLV-pandorabox** [comment]
> I think he found this on pandorabox, meta repo at: https://github.com/pandorabox-io/pandorabox-mods
> 
> Edit: game is default MTG

Yes that is it . Also I should mention that you can't see yourself sitting in 3rd person view, but a second player can.

**Comment by MCLV-pandorabox** [comment]
I'm sorry to bother all y'all, but I'm seeing more and more players floating around sitting and personally all the fun and utility has seeped out of using a chair. Is anything going to be done about this in the near future?
I mean I really appreciate what wsor tried to do here. I actually saw that 'bug' years ago and wanted to fix it too. But I soon realized what a hornets nest I'd be getting into. I admire your bravery, but can we just please revert back to what it was like until you perhaps can come up with something that automatically changes the users 'sit' animation when they move away from the chair. 
And fix the sit height also? I really don't enjoy my hitbox being stuck in the y+3 node above the chair anytime I sit down.

I'm sorry if this comes over as a little bit whiny but I think it's a rather unnecessary change and it shouldn't take a month to fix this.
Please just revert the changes to the chairs.

Thanks for all your hard work, it truly is appreciated

**Comment by wsor4035** [comment]
waiting on https://github.com/mt-mods/homedecor_modpack/tree/player_monoids - blame physics modifiers bad design in the engine, and in this specific case: https://github.com/pandorabox-io/gravity_manager/blob/79adb2ccfc510b4ab552dd175da892894fb30966/timer.lua

waiting back to hear from niklp on the status of that branch

**Comment by MCLV-pandorabox** [comment]
Awesome! Progress.
However I think you just made it a little worse.

Sitting position still too high. Getting hurt by whatever is on chair.y+2.
That killed me. And afer that I remained sitting after respawn and also completely unable to move.

Luckily I had a chair nearby I could click so I could move again

**Comment by MCLV-pandorabox** [comment]
Entity hell seems to work better. Thanks
Still getting hurt by dangerous node on chair.y+2
Still able to move around while seated, but I have to die on a chair for that one now.

chair.y+2
I mean, if you put a simple homedecor:armchair on position 0, then put default:stone on position 1 above that. That is obviously a big outchy. But if you place another default:stone on top of that (position 2 eg chair.y+2) and remove the stone on position 1. you should have enough clearance and not be hurt. This was the case before you introduced the new seating arrangement. and a lot of in-game builds kinda depend on that behavior I think. But I don't know many other servers than Pandorabox

**Comment by wsor4035** [comment]
https://github.com/mt-mods/homedecor_modpack/commit/d8ae7a360da6204f6bb84d719dd4eeee5df3b6db for catching when a player dies. make an issue to kep track of it please about the "chair.y+2", dont really care/low priority currently
